src/main/java/com/spotride/spotride/user/UserMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.spotride.spotride.user;
+
+import com.spotride.spotride.user.dto.UserRequestDto;
+import com.spotride.spotride.user.dto.UserResponseDto;
+import com.spotride.spotride.user.model.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * Mapper for {@link User}.
+ */
+@Mapper
```

Comment:
Check how to make it component.

Spring takes responsibilty how to make it singletone. Otherwise it is a bad practice